pyosmapi/osm_api.py

```diff
@@ -10,18 +10,22 @@
 
 class OsmApi:
     def __init__(self, instance: str = "dev"):
+        self.url_extension = '/api/0.6'
         if instance.lower() == "main":
             self.base_url = DEFAULT_OSM_URL
+            logger.info('Using osm main api')
         elif instance.lower() == "dev":
             self.base_url = DEFAULT_OSM_DEV_URL
+            logger.info('Using osm dev api')
         else:
-            self.base_url = instance
+            logger.warning('Key not found, using osm dev api..')
+            self.base_url = DEFAULT_OSM_DEV_URL
```

Comment:
Why did you change this?
The point of my pull request is that if you set a URL instead of `dev` or `main` that is used as the API URL, meaning you can use other API instances.